jans-config-api/plugins/user-mgt-plugin/src/main/java/io/jans/configapi/plugin/mgt/service/UserMgmtService.java

```diff
@@ -443,4 +443,8 @@ private String getDnForAttribute(String inum) {
         }
         return String.format("inum=%s,%s", inum, attributesDn);
     }
+
+    private static boolean isEmptyOrNullCollection(Collection<?> collection) {
+        return (collection == null || collection.isEmpty());
+    }
```

Comment:
Approving but in future you can re-use `org.apache.commons.collections.CollectionUtils#isEmpty`